website/docs/r/dns_custom_resolver.html.markdown

```diff
@@ -43,11 +39,6 @@ Review the argument reference that you can specify for your resource.
 - `instance_id` - (Required, String) The GUID of the private DNS service instance.
 - `name`- (Required, String) The name of the custom resolver.
 - `description` - (Optional, String) Descriptive text of the custom resolver.
-- `locations`- (Required, Set) The list of locations where this custom resolver is deployed. There is no update for location argument in resolver resource.
```

Comment:
please donot remove.. it is made as optional.. feature is not removed from resource

Reply:
Updated documents